src/ws/client.rs

```diff
@@ -145,6 +146,7 @@ impl Client {
             origin: None,
             protocols: None,
             max_size: 65_536,
+            max_continuation_size: 1_048_576,
```

Comment:
Why continuation frame has such big size?
I'm not very well familiar with websocket protocol, but I'd expect it to be few times more than default `max_size` but 1mb seems quite big as default.

Though I'm not sure what is common practice, but maybe something around `256kb` would be better as default?
But I don't feel strongly about it, so I leave it to you since Nikolay is ok with it